src/main/java/net/dv8tion/jda/internal/handle/ChannelUpdateHandler.java

```diff
@@ -713,29 +718,39 @@ private void handleTagsUpdate(ForumChannelImpl channel, DataArray tags)
                     String name = tagJson.getString("name");
                     boolean moderated = tagJson.getBoolean("moderated");
 
-                    // TODO: Events?
-                    impl.setPosition(i);
-                    impl.setEmoji(tagJson);
                     String oldName = impl.getName();
+                    EmojiUnion oldEmoji = impl.getEmoji();
+
+                    impl.setEmoji(tagJson);
+
+                    impl.setPosition(i);
```

Comment:
What _is_ position and why aren't we eventing on it?

Reply:
So position controls the order of emojis. Why don't we event on this?
Also, do we need to provide a way to rearrange the order of tags like we do roles?

Reply:
Good enough for me. We can add it if it is needed later